src/linagora.esn.calendar/app/event-view/external-user/event-view-external-user.controller.js

```diff
@@ -35,9 +35,10 @@ const _ = require('lodash');
       });
 
       $http({ method: 'GET', url: self.linksMapping[partstat] }).then(function() {
-        var participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
+        const participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
+        const translatedParticipation = esnI18nService.translate(participation);
```

Comment:
Is there always a translation for `participation`?

Reply:
yes , there is